src/resources/js/tickets.js

```diff
@@ -466,12 +389,18 @@ var ticketHeaderImage = window.ticketHeaderImage || {};
 					}
 
 					// Settings table
-					if ( 'undefined' !== response.data.capacity_table ) {
+					if ( 'undefined' !== typeof response.data.capacity_table ) {
 						$( document.getElementById( 'tribe_expanded_capacity_table' ) ).replaceWith( response.data.capacity_table );
 					}
 
+					// Settings table
+					if ( 'undefined' !== typeof response.data.settings_panel ) {
+						$( document.getElementById( 'tribe_panel_settings' ) ).replaceWith( response.data.settings_panel );
+						$settings_panel = $( document.getElementById( 'tribe_panel_settings' ) );
```

Comment:
Rather than doing the same `document.getElementById( 'tribe_panel_settings' )` call twice, just do it once and assign into a variable.